.github/workflows/ext_run_test_integration.yml

```diff
@@ -0,0 +1,33 @@
+name: external_run_test-integration
+
+run-name: ${{ inputs.prefix }}_${{github.workflow}}_${{ inputs.uuid }}
+
+on:
+  workflow_dispatch:
+    inputs:
+      prefix:
+        description: 'Prefix runtime workflow'
+        required: false
+        type: string
+        default: ""
+      uuid:
+        description: 'UUID that can correct understand start'
+        required: false
+        type: string
+        default: ""
+      clickhouse_version:
+        description: 'Clickhouse version'
+        required: true
+        type: string
```

Comment:
Why do we need `prefix` and `uuid` parameters? Is it not enough to have only `clickhouse_version`?

Reply:
It seems we can use `inputs.clickhouse_version` in `run-name` and get rid of `input.prefix`.

Also suggest to make id parameter more abstract - just `id` without any implications on format.

```
run-name: ${{ github.workflow }}_${{ inputs.clickhouse_version }}_${{ inputs.id || github.run_number }}

on:
  workflow_dispatch:
    inputs:
      clickhouse_version:
        description: 'ClickHouse version'
        required: true
        type: string
      id:
        description: 'Run identifier'
        required: false
        type: string
        default: ""
```
?

Reply:
Yes, it is possible. I will change Tasklet config in out side too. It is no problem. I will change.

Reply:
I need time for think. Because this formula of "run-name" must me used in our side in Tasklets. Tasklets will be universal. May be I will add optional field in Tasklet for custom.

Reply:
Corrected.